krabs/krabs/schema.hpp

```diff
@@ -42,17 +42,18 @@ namespace krabs {
 
         /**
          * <summary>
-         * Constructs a schema from an event record instance.
+         * Constructs a schema from an event record instance
+         * using the provided schema_locator.
          * </summary>
          *
          * <example>
-         *   void event(const EVENT_RECORD &record)
+         *   void event(const EVENT_RECORD &record, const krabs::trace_context &trace_context)
```

Comment:
```suggestion
         *   void on_event(const EVENT_RECORD &record, const krabs::trace_context &trace_context)
```

Reply:
I'm okay with checking this in to `master`. If we're making an API-breaking change then let's bump the major version, otherwise this can be a dot release.